docs/user-guide/story-setting.md

```diff
@@ -0,0 +1,19 @@
+ 
+ 
+For setting the information that appears on the story top bar on the *View Mode* the user need to go back on the *Edit Mode* and open the **Setting** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
+
+<img src="../img/story-setting/story-setting.jpg" class="ms-docimage"/>
+
+The user can enable/disable the *Title*, the *Icon* and the *Navigation bar* and can :
```

Comment:
The editor can enable or disable and edit the following components:

- The *Title* of the story, the default value is the title given to the story's resource in MapStore 
- The story *Logo*, that can be for example an image that represents your organization or something connected to the story itself.
- The *Navigation Toolbar* to improve the story navigation for end users. Each section of the story is reported in a tree and the editor can establish which section should appear in the toolbar to allow end users to quickly navigate the story.